regions/shapes/annulus.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import math
+import operator
+
+import numpy as np
+
+from astropy import wcs
+from astropy import coordinates
+from astropy import units as u
+
+from ..core import CompoundPixelRegion, CompoundSkyRegion
+from ..shapes import CirclePixelRegion, CircleSkyRegion
+
+
+class AnnulusPixelRegion(CompoundPixelRegion):
```

Comment:
This should be called `CircleAnnulusPixelRegion` because an annulus can have any shape, e.g. we define elliptical and rectangular annulus apertures in photutils.